src/ppm.cpp

```diff
@@ -845,55 +844,55 @@ int main( int argc, char* argv[] )
-    ppm.addOption( 'f', "filtered-topic", "The unfiltered produce topic.", true );
-    ppm.addOption( 'p', "partition", "Consumer topic partition from which to read.", true );
-    ppm.addOption( 'g', "group", "Consumer group identifier", true );
-    ppm.addOption( 'b', "broker", "List of broker addresses (localhost:9092)", true );
-    ppm.addOption( 'o', "offset", "Byte offset to start reading in the consumed topic.", true );
-    ppm.addOption( 'x', "exit", "Exit consumer when last message in partition has been received.", false );
-    ppm.addOption( 'd', "debug", "debug level.", true );
-    ppm.addOption( 'm', "mapfile", "Map data file to specify the geofence.", true );
-    ppm.addOption( 'v', "log-level", "The info log level [trace,debug,info,warning,error,critical,off]", true );
-    ppm.addOption( 'D', "log-dir", "Directory for the log files.", true );
-    ppm.addOption( 'R', "log-rm", "Remove specified/default log files if they exist.", false );
-    ppm.addOption( 'i', "log", "Log file name.", true );
-    ppm.addOption( 'h', "help", "print out some help" );
+    ppm.addOption('c', "config", "Configuration for Kafka and Privacy Protection Module.", true);
+    ppm.addOption('C', "config-check", "Check the configuration and output the settings.", false);
+    ppm.addOption('u', "unfiltered-topic", "The unfiltered consume topic.", true);
+    ppm.addOption('f', "filtered-topic", "The unfiltered produce topic.", true);
+    ppm.addOption('p', "partition", "Consumer topic partition from which to read.", true);
+    ppm.addOption('g', "group", "Consumer group identifier", true);
+    ppm.addOption('b', "broker", "List of broker addresses (localhost:9092)", true);
+    ppm.addOption('o', "offset", "Byte offset to start reading in the consumed topic.", true);
+    ppm.addOption('x', "exit", "Exit consumer when last message in partition has been received.", false);
+    ppm.addOption('d', "debug", "debug level.", true);
+    ppm.addOption('m', "mapfile", "Map data file to specify the geofence.", true);
+    ppm.addOption('v', "log-level", "The info log level [trace,debug,info,warning,error,critical,off]", true);
+    ppm.addOption('D', "log-dir", "Directory for the log files.", true);
+    ppm.addOption('R', "log-rm", "Remove specified/default log files if they exist.", false);
+    ppm.addOption('i', "log", "Log file name.", true);
+    ppm.addOption('h', "help", "print out some help");
 
     if (!ppm.parseArgs(argc, argv)) {
         ppm.usage();
-        exit( EXIT_FAILURE );
+        exit(EXIT_FAILURE);
     }
 
     if (ppm.optIsSet('h')) {
         ppm.help();
-        exit( EXIT_SUCCESS );
+        exit(EXIT_SUCCESS);
     }
 
     // can set levels if needed here.
-    if ( !ppm.make_loggers( ppm.optIsSet('R') )) {
-        exit( EXIT_FAILURE );
+    if (!ppm.make_loggers(ppm.optIsSet('R'))) {
+        exit(EXIT_FAILURE);
     }
 
     // configuration check.
     if (ppm.optIsSet('C')) {
```

Comment:
In the ppm.cpp and kafka_consumer.cpp file, can we add the confluent cloud integration configuration parameter to the docker-compose.yml and sample.env file? or wherever the suitable place for setting the environment variables: CONFLUENT, CONFLUENT_KEY, CONFLUENT_SECRET, KAFKA_TYPE etc... 
- If you have time, please update the document how to get the KEY and SECRET from the API mentioned in the readme.md?